KubeArmor/policy/policy.go

```diff
@@ -38,12 +47,22 @@ func (p *ServiceServer) ContainerPolicy(c context.Context, data *pb.Policy) (*pb
 
 // HostPolicy accepts host policy event on gRPC service and updates host security policies. It responds with 1 if success else 0.
 func (p *ServiceServer) HostPolicy(c context.Context, data *pb.Policy) (*pb.Response, error) {
+	
 	policyEvent := tp.K8sKubeArmorHostPolicyEvent{}
 	res := new(pb.Response)
 
 	err := json.Unmarshal(data.Policy, &policyEvent)
 	if err == nil {
+
+		if policyEvent.Object.Metadata.Name != ""{
+
 		p.UpdateHostPolicy(policyEvent)
+
+		}else{
+
+		kg.Warn("Empty Host Policy Event")
+
+		}
```

Comment:
Let's add res,Status = 0 as well in the else condition and move the status = 1 to the if condition. Also I would suggest you to run `gofmt` once it will handle the formating for ya!

Reply:
Okay thanks

Reply:
okay thanks I have done that